language/e2e-testsuite/src/tests/admin_script.rs

```diff
@@ -65,6 +65,71 @@ fn admin_script_rotate_key_single_signer() {
     executor.new_block();
     let output = executor.execute_and_apply(txn);
 
+    // The transaction should not trigger a reconfiguration.
+    let new_epoch_event_key = libra_types::on_chain_config::new_epoch_event_key();
+    assert!(!output
+        .events()
+        .iter()
+        .any(|event| *event.key() == new_epoch_event_key));
+
+    let updated_sender = executor
+        .read_account_resource(new_account.account())
+        .expect("sender must exist");
+
+    assert_eq!(updated_sender.authentication_key(), new_key_hash.as_slice());
+}
+
+#[test]
+fn admin_script_rotate_key_single_signer_new_epoch() {
+    let mut executor = FakeExecutor::from_genesis_file();
+    let new_account = AccountData::new(100_000, 0);
+    executor.add_account_data(&new_account);
+
+    let privkey = Ed25519PrivateKey::generate_for_testing();
+    let pubkey = privkey.public_key();
+    let new_key_hash = AuthenticationKey::ed25519(&pubkey).to_vec();
+
+    let script_body = {
+        let code = "
+    import 0x1.LibraAccount;
+    import 0x1.LibraConfig;
+
+    main(lr_account: &signer, account: &signer, auth_key_prefix: vector<u8>) {
+      let rotate_cap: LibraAccount.KeyRotationCapability;
+      rotate_cap = LibraAccount.extract_key_rotation_capability(copy(account));
+      LibraAccount.rotate_authentication_key(&rotate_cap, move(auth_key_prefix));
+      LibraAccount.restore_key_rotation_capability(move(rotate_cap));
+
+      LibraConfig.reconfigure(move(lr_account));
+      return;
+    }
+";
+
+        let compiler = Compiler {
+            address: account_config::CORE_CODE_ADDRESS,
+            extra_deps: vec![],
+            ..Compiler::default()
+        };
+        compiler
+            .into_script_blob("file_name", code)
+            .expect("Failed to compile")
+    };
```

Comment:
some alignment weirdness here?

Reply:
I don't think so since this is what `cargo x fmt` spits out? The alignment makes sense to me since this is closing the `let` on line 92.